pkg/operator/guardbudget/guard_budget_controller.go

```diff
@@ -106,9 +119,23 @@ func (c *GuardBudgetController) checkGuardBudget() error {
 		deploymentScaledDown = true
 	}
 
+	clusterVersion, err := c.clusterVersionLister.Get("version")
+	if err != nil {
+		return err
+	}
+	for _, override := range clusterVersion.Spec.Overrides {
```

Comment:
ah, yeah, we don't offer that option anymore. and you don't need delete powers

Reply:
removed